### DISCUSSION TITLE
When is 0.8.0 expected to be released

### DISCUSSION THREAD

**Discussion by vsraptor** [goodmami/wn#132] [Q&A] [answered]
i'm anticipating the new Similarity measures ;) ...

**Comment by goodmami** [comment, marked as answer]
When it's done? :grin: 

Unfortunately, Wn isn't my priority this month so progress has been a bit slow. I have pushed what I have been working on to the v0.8.0 branch and opened a PR (#128), so feel free to take a look. I think all of the issues in the [milestone](https://github.com/goodmami/wn/milestone/5) have been accomplished, but mainly what's missing now is tests for the new similarity metrics.

As usual, PRs are welcome.

**Reply by goodmami** [reply]
Released: https://github.com/goodmami/wn/releases/tag/v0.8.0
